### PR TITLE
Fix a bug for multilingual ASR

### PR DESCRIPTION
**Pull request by csukuangfj**
We should use
```
logits[:, -1, :]
```
for the `sot_sequence`, not `logits[:, 0, :]`.

### CODE REVIEW

**Comment by csukuangfj** [comment]
Thanks to @zzzacwork  for pointing out this bug.
